apps/explorer/src/explorer/components/TransanctionBatchGraph/index.tsx

```diff
@@ -32,9 +32,13 @@ import { usePrevious } from '../../../hooks/usePrevious'
 import { Network } from '../../../types'
 import { DropdownOption, DropdownPosition } from '../common/Dropdown'
 
+// eslint-disable-next-line react-hooks/rules-of-hooks
 Cytoscape.use(popper)
+// eslint-disable-next-line react-hooks/rules-of-hooks
 Cytoscape.use(noOverlap)
+// eslint-disable-next-line react-hooks/rules-of-hooks
 Cytoscape.use(fcose)
+// eslint-disable-next-line react-hooks/rules-of-hooks
 Cytoscape.use(klay)
```

Comment:
What's the rule being broken?

Reply:
Now this call is considered as a React hook call which is supposed to be called only in components.
I know, it's weird, but since this is the only place, I just muted on the place.